styles.css

```diff
@@ -21,6 +21,19 @@
             "footer";
 }
 
+@media only screen and (min-width: 992px) {
```

Comment:
Ik zou in dit geval min-width 576px aanhouden. Zie ook de studiehandleiding voor de media breakpoints die novi adviseert.

Reply:
Ok, die studiehandleiding er maar eens bijpakken..